lib/cisco_node_utils/vrf.rb

```diff
@@ -73,5 +73,25 @@ def shutdown=(val)
     rescue Cisco::CliError => e
       raise "[vrf #{@name}] '#{e.command}' : #{e.clierror}"
     end
+
+    # Vni (Getter/Setter/Default)
+    def vni
+      config_get('vrf', 'vni', @args)
+    end
+
+    def vni=(id)
+      if id == default_vni
+        @args[:state] = 'no'
+        @args[:id] = vni
+      else
+        @args[:state] = ''
+        @args[:id] = id
+      end
+      config_set('vrf', 'vni', @args)
```

Comment:
Don't use `@args` in this setter -- code this setter like the other setters in this file:

```
  def vni=(id)
    no_cmd = (id) ? '' : 'no'
    config_set('vrf', 'vni', vrf: @name, state: no_cmd, id: id)
  rescue Cisco::CliError => e
    raise "[vrf #{@name}] '#{e.command}' : #{e.clierror}"
  end
```